testing/mockeryutils/Makefile

```diff
@@ -1,25 +1,20 @@
-#  Copyright 2024 The Nephio Authors.
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
-.PHONY: all
-all: fmt test
+IMAGE_TAG ?= latest
+REGISTRY ?= docker.io/nephio
+IMAGE_NAME ?= ipam-fn
```

Comment:
Is this correct? Do we need a Makefile here?

Reply:
Yes, if it's not here the build fails. The main Makefile expects a Makefile in every submodule that contains "go" files.

Reply:
Thanks @efiacor , I used this Makefile